src/Microsoft.AspNet.Mvc.Xml/SerializableErrorWrapper.cs

```diff
@@ -0,0 +1,57 @@
+﻿using System;
+using System.Xml;
+using System.Xml.Schema;
+using System.Xml.Serialization;
+
+namespace Microsoft.AspNet.Mvc.Xml
+{
+    /// <summary>
+    /// Wrapper class for <see cref="SerializableError"/> to enable it to
+    /// be serialized by the xml formatters.
+    /// </summary>
+    [XmlRoot("Error")]
+    public sealed class SerializableErrorWrapper : IXmlSerializable
```

Comment:
This does not have unit tests yet...will write them and update this PR...wanted to get some initial input...